bitvision/bitvision.py

```diff
@@ -15,126 +15,160 @@
 CONFIG = configparser.ConfigParser()
 CONFIG.read("config.ini")
 
-
 #########
 # HELPERS
 #########
 
+# BitVision Command Line Trading Platform
+# Aaron Lichtman and Jon Shobrook
 
-def login():
+import os
+import sys
+import urwid
+
+# Place Your Own Trade is not shown until Trading Deactivated.
+choices = ["Activate/Deactivate Automated Trading", "Deposit/Withdraw Money", "Place Your Own Trade"]
+
+
+def menu(title, choices):
+	"""
+    Make a menu with choices displayed.
     """
+	body = [urwid.Text(title), urwid.Divider()]
+	for c in choices:
+		button = urwid.Button(c)
+		urwid.connect_signal(button, 'click', item_chosen, c)
+		body.append(urwid.AttrMap(button, None, focus_map='reversed'))
+	return urwid.ListBox(urwid.SimpleFocusListWalker(body))
+
+
+def item_chosen(button, choice):
+	response = urwid.Text([u'You chose ', choice, u'\n'])
+	done = urwid.Button(u'Ok')
+	urwid.connect_signal(done, 'click', exit_program)
+	main.original_widget = urwid.Filler(urwid.Pile([response,
+	                                                urwid.AttrMap(done, None, focus_map='reversed')]))
+
+
+def exit_program(button):
+	raise urwid.ExitMainLoop()
+
+
+def login():
+	"""
     Prompts the user for their GDAX API key, secret, and passphrase before
     logging into the trading system.
 
     @return: authenticated GDAX Client object
     """
 
-    # TODO: Add a helper message with GDAX instructions and a security disclaimer
+	# TODO: Add a helper message with GDAX instructions and a security disclaimer
```

Comment:
Not sure, but I think you may have added extra indentation here by mistake. Haven't verified in my editor though.

Reply:
Looks fine on my editor. Not sure.

Reply:
Looks like it was fixed in this most recent commit.